simple/run.sh

```
@@ -1,21 +1,48 @@
+CSV=$DIR/../stats.csv
+REPORT=$DIR/../report.lua
 NODE=node@`node -v`
+LOG=$DIR/logs
+rm -rf $LOG
+mkdir -p $LOG
 
 echo
-EGG_SERVER_ENV=prod node $NODE_FLAGS `dirname $0`/dispatch.js $1 &
-pid=$!
+# koa2
+EGG_SERVER_ENV=prod node $NODE_FLAGS $DIR/dispatch.js koa2 7001 2>&1 > $LOG/koa2.log &
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)